<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialPortDialog</class>
 <widget class="QDialog" name="SerialPortDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>275</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>240</width>
    <height>275</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>240</width>
    <height>275</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Serial Port Settings</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>201</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QComboBox" name="baudRateComboBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="portNameComboBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelPortName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>22</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Port:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelBaudRate">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>54</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Baudrate:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="dataBitsComboBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="parityComboBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="stopBitsComboBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="flowControlComboBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelDataBits">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>83</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Data Bits:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelParity">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>113</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Parity:</string>
   </property>
  </widget>
  <widget class="QLabel" name="stopBitsLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>143</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop Bits:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelFlowControl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>173</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Flow Control:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SerialPortDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SerialPortDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
